modules/Configurator/tpls/EditView.tpl

```diff
@@ -372,15 +372,29 @@
 	</tr>
 
 
-
 </table>
+	<th scope="row" colspan="6"><h4>{$MOD.LBL_SENTRY}</h4></th>
+	<table width="100%" border="0" cellspacing="1" cellpadding="0" class="edit view">
+		<td scope="row">{$MOD.LBL_SENTRY_ENABLED}:
+			{if !empty($config.sentry.enabled)}
+				{assign var='sentry_checked' value='CHECKED'}
+			{else}
+				{assign var='sentry_checked' value=''}
+			{/if}
 
-<table  width="100%" border="0" cellspacing="1" cellpadding="0" class="edit view">
-{if $logger_visible}
-<tr>
-<th align="left" scope="row" colspan="6"><h4>{$MOD.LBL_LOGGER}</h4></th>
-</tr>
-	<tr>
+		<td><input type='hidden' name='sentry_enabled' value='false'><input name='sentry_enabled' type="checkbox"
```

Comment:
I think the formatting here got screwed up?

Reply:
Yeah, didn't actually check what this looked like or even if it worked, just threw it into the draft so I wouldn't forget to make this configurable somewhere in the front-end :P.